server/src/main/java/org/apache/iotdb/db/metadata/MTree.java

```diff
@@ -964,7 +987,13 @@ private void findPath(MNode node, String[] nodes, int idx, String parent,
     if (node.isLeaf()) {
       if (nodes.length <= idx) {
         String fileName = node.getDataFileName();
-        String nodePath = parent + node;
+        String nodeName;
+        if(node.getName().contains(TsFileConstant.PATH_SEPARATOR)){
+          nodeName = "\"" + node + "\"";
+        }else{
+          nodeName = "" + node;
+        }
+        String nodePath = parent + nodeName;
```

Comment:
Please format these codes. btw, please check the code format of other files.

Reply:
Thanks for your reminding! The code is well formatted now.